drivers/i2s/Kconfig.nrfx

```diff
@@ -22,3 +22,25 @@ config I2S_NRFX_TX_BLOCK_COUNT
 	default 4
 
 endif # I2S_NRFX
+
+menuconfig TDM_NRFX
+	bool "nRF TDM nrfx driver"
```

Comment:
I would prefer to avoid nrfx name here, because shim implementation uses mostly hal layer of the nrfx